lib/llvm-backend/src/code.rs

```diff
@@ -1533,7 +1569,18 @@ impl<'ctx> FunctionCodeGenerator<CodegenError> for LLVMFunctionCodeGenerator<'ct
                 }
 
                 let frame = state.outermost_frame()?;
-                builder.build_unconditional_branch(*frame.br_dest());
+
```

Comment:
Can you add a comment before this statement to give context?

```suggestion
                        // We remove unneeded basic blocks, in case is empty
```